index.html

```diff
@@ -21,21 +21,31 @@
   <!-- The header section for mobile -->
   <header>
     <a href="#" class="logo">Grace Abike</a>
-    <nav class="mobile">
-      <img src="./imgs/Menumobile.png" alt="menu">
-    </nav>
+    <div id="hamburger-icon" class="mobile" onclick="toggleMenu(this)">
+      <div id="bar1" class="bar"></div>
+      <div id="bar2" class="bar"></div>
+      <div id="bar3" class="bar"></div>
+      <div id="mobileNav" class="overlay mobile mobile-menu">
+        <div class="overlay-content">
+          <a class="bar-link1" href="#">Portfolio <span class="heavy-right">&#x276F;</span></a>
+          <a class="bar-link2" href="#">About <span class="heavy-right">&#x276F;</span></a>
+          <a class="bar-link3" href="#">Contact <span class="heavy-right">&#x276F;</span></a>
+        </div>
```

Comment:
- [ ] As requested by the previous reviewer, When users click any of the menu options they should redirect to the correct section of the webpage. When I click on the menu options it doesn't take me to anywhere because the links you are using are empty. Please fix this issue and use the correct `IDs` of the sections in the links.

Reply:
@thecodechaser thanks for your comments I really appreciate,
I implemented the link before
<img width="595" alt="image" src="https://user-images.githubusercontent.com/88346506/188505750-9cedeead-17b1-4b78-b4c5-9f07ea6c2d11.png">